src/main/scala/de/htwg/se/sudoku/model/playerComponent/playerMicroImpl/PlayerHttpServer.scala

```diff
@@ -0,0 +1,43 @@
+package de.htwg.se.sudoku.model.playerComponent.playerMicroImpl
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.stream.ActorMaterializer
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+class PlayerHttpServer(player: Player) {
+
+  implicit val system: ActorSystem = ActorSystem("system")
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  implicit val executionContext: ExecutionContextExecutor = system.dispatcher
+
+  val route: Route = get {
+    pathPrefix("sudoku" / "model" / "player") {
+      path("v1" / "string") {
+        complete(StatusCodes.OK -> player.name)
+      }
+    } ~
+      post {
+        pathPrefix("sudoku" / "model" / "player" / "v1") {
+          path(Segment) { name => {
+            Player(name)
+            complete(StatusCodes.OK)
+          }
+          }
+        }
+      }
+  }
+
+  val bindingFuture: Future[Http.ServerBinding] = Http().bindAndHandle(route, "localhost", 8083)
+
+  def unbind(): Unit = {
+    bindingFuture
+      .flatMap(_.unbind())
+      .onComplete(_ => system.terminate())
```

Comment:
I'm not sure about this, but all files (e.g. (FileIoHttpServer)[https://github.com/bb30/de.htwg.se.SudokuInScala/pull/11/files#diff-20cefbb470cec236cf9d2bec1a69f07aR15]) initialize the `ActorSystem` with param `system` and I think it is possible this will just create one instance of the `ActorSystem`. If so, terminating it in every `unbind` could have unwanted consequences (i.e. terminating it twice might result in an exception?).

Reply:
Hmm good point, I have to think about this.

Reply:
Should be fixed with https://github.com/bb30/de.htwg.se.SudokuInScala/pull/11/commits/7931f54c7daeae1cfafc8c6a140a606ea81996f6